### DISCUSSION TITLE
starting/ending backup alledgedly failing each time

### DISCUSSION THREAD

**Discussion by greenemso** [iterate-ch/cyberduck#15340] [Q&A]
error message appears at end of uploading large files 5-15GB as follows [but when i press cancel its downloaded and showing as copy complete in transfer dialogue detail box] - so not really an error.

Error (192.168.0.6 - SFTP)
One or more of the specified parts could not be found. The part may not have been uploaded, or the specified entity tag may not match the part's entity tag.:
PartNumber=null, MinSizeAllowed=null,
ProposedSize=null, ETag=null.

also error when starting backup each time as follows.. (but when i just click 'try again' and it just works every time) - so again not really an error, but just annoying.

Failure to read attributes of vault.cryptomator. 

nodename nor servname provided, or not known. DNS is the network service that translates a server name to its Internet address. This error is most often caused by having no connection to the Internet or a misconfigured network. It can also be caused by an unresponsive DNS server or a firewall preventing access to the network.

**Comment by dkocher** [comment]
Can you please [enable debug logging](https://docs.cyberduck.io/cyberduck/support/#debug-logging-level) in _Preferences → Connection_  and attach the log file.
